llzk-sys/build-support/src/mlir.rs

```diff
@@ -26,20 +26,27 @@ impl<'a> MlirConfig<'a> {
         }
     }
 
-    pub fn mlir_path(&self) -> Result<Cow<'static, Path>> {
-        let path = Path::new(env!("MLIR_SYS_200_PREFIX"));
+    pub fn mlir_path(&self) -> Result<Option<Cow<'static, Path>>> {
+        let Some(path) = option_env!("MLIR_SYS_200_PREFIX").map(Path::new) else {
```

Comment:
The README.md in `llzk-sys` still refers to MLIR 18, so I think that needs to be updated as well.